web/custom-plugins/forecast-of-response/src/utils/createContainerJob.ts

```diff
@@ -33,6 +34,7 @@ export function createContainerJob(
       `--workflow=${workflow}`,
       `--input=${DEFAULT_DATASOURCE_ID}/${inputId}`,
       `--target=${targetFolder}`,
+      `--operation-results-dotted-id=${result_dotted_id}`,
```

Comment:
The SRE wrapper should support only one way of defining a upload target. Looks like we are going for "dottedPath" and not "folder-path".